### ISSUE TITLE
ButtonLink component doesn't "see" navigation.redirect

### ISSUE THREAD

**Issue by mathieunicolas** [nuxt-themes/docus#749]
Well, last version of Docus.
When I click on the top navbar on a folder, it "opens" the navigation.redirect option.
When I click on a ButtonLink which points to the exact same URL, it "opens" the index file without taking the navigation.redirect option.

Is it a normal thing ? How could I make it work like the top navbar item ? I checked in the Vue devtools and I see a NuxtLink component. I thought it was because of some kind of "normal" link but appeartently not !

**Comment by mathieunicolas** [comment]
I updated my issue, I'm sorry I didn't take the time to explain as I should have to in the first draft.

**Comment by Tahul** [comment]
@farnabaz any idea about this one?

I need to dig into it, but it seem to be related to Content?

**Comment by farnabaz** [comment]
I agree that there is a bit of ambiguity and conflict between `redirect` and `navigation.redirect`

- `redirect` uses when user navigates to the page 
- `navigation.redirect` uses to determine nav item link

So in `test-c`:
- inside nav bar the link will use `navigation.redirect` and it will not care about `redirect`
- inside button, when user hit the button, it will directly go to `test-c` then in the page it will use `redirect` and will not care about `navigation.redirect`

Let's discuss about the perfect behaviour and combination of these two before changing anything.

WDYT @Tahul ?

**Comment by mathieunicolas** [comment]
I don't know if I can help, but my usecase is simple : with Docus I make a website I use with my students. At the beginning of each hour, they go the the website, then click on their class, and I'd like to define a redirect to the "page of the day". 
That's why I also want to be able to collapse all the nodes in the asideTree except the one that contains the page of the day. That's why I also made an issue related to useDocus(), that may overwrite the collapsed state of the asideTree nodes. 

Let me know if I can help more !
